_posts/2020-08-26-engineering-firstweek.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "My first week at Scribd engineering"
+tags:
+- scribd
+team: Core Infrastructure
+author: zherner
+---
+
+
+My first week at Scribd engineering!
+
+Hello everyone, I'm new at Scribd and here's what the first week is like on the Core Infrastructure engineering team. It's full of smart people, great culture, and cool tech and I wanted to share my experience.
+
+The journey starts of course, with an interview phase. This round has to be one of the most pleasant interview experiences I've had ….I've been through the wringer before. From start to finish I was clear on next steps and what to expect. The recruiter was awesome, she scheduled prep meetings with me to answer questions before interviews. I got to meet my manager, the team director, all but one of my teammates (he was on PTO) and folks on other teams, even teammates in Amsterdam. The process was entirely remote due to the pandemic but that didn't take away from the experience. To me it felt just as personal as an onsite would have been by making excellent use of Google meet.
+
+Thankfully I'm not alone at first day orientation, I'm joined by a group of new hires. Since we are all in the same boat we've started our own new-hire slack channel to support each other. To get started there is a helpful and thorough checklist for us newbies to walkthrough for setting up the basics. The better part is the team specific onboarding guide that is focused on setup tasks that pertain to tools I'll use for everyday type work. The best part though, has got to be that I got a new buddy! Scribd does new-hire buddies to show you the ropes. It goes a long way to take a little bit of the new-job tension off. You can go to your new hire buddy for anything from onboarding to team specific questions. In my case my buddy is a tech lead so they have a wealth of knowledge on our stack or knows who to point you to if it isn't in their domain.
```

Comment:
`my buddy is a tech lead so they have a` "he/she" feels more natural here than "they", but up to you 😀

Reply:
Agreed. Got it in.